### PR TITLE
#778: Rename AtlasSDK framework to ZalandoCommerceAPI

### PR DESCRIPTION
**Pull request by bonkey**
Dependencies not updated, yet (another PR)

### CODE REVIEW

File: Sources/ZalandoCommerceAPI/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# AtlasSDK
+# Zalando Commerce API iOS Framework
 
 [Configuration](https://github.com/zalando-incubator/atlas-ios/wiki/Configuration)
```
Comment (hani-ibrahim) [inline Sources/ZalandoCommerceAPI/README.md]: will you rename the project name on Github too? `https://github.com/zalando-incubator/atlas-ios`

**Comment by hani-ibrahim** [comment]
👍

File: Sources/ZalandoCommerceAPI/README.md (hunk 1 above)
Comment (bonkey) [inline Sources/ZalandoCommerceAPI/README.md]: Yes, this is the change I'm afraid the most, actually.

Comment (hani-ibrahim) [inline Sources/ZalandoCommerceAPI/README.md]: 3o7abrh8o4hmgeav9e

**Pull request merged**